contributing_to_docs/term_glossary.adoc

```diff
@@ -233,6 +233,17 @@ When referencing the actual object, write as `Event`. See link:doc_guidelines.ad
 // NOTE: This is inconsistently used, e.g. https://docs.openshift.com/container-platform/4.5/rest_api/metadata_apis/event-core-v1.html
 See: link:https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#event-v1-core[Event v1 core API], link:https://github.com/cloudevents/spec/blob/master/primer.md#cloudevents-concepts[CloudEvents concepts], and link:https://github.com/cloudevents/spec/blob/master/spec.md#event[CloudEvents specification].
 
+== G
+
+''''
+=== Group Version Kind (GVK)
```

Comment:
I feel like there are a million different ways to write this, and I honestly don't have a strong opinion on if it should "Group Version Kind" or "group version kind" or "GroupVersionKind" or "group/version/kind". I see it so many different ways that I'm not sure which is really correct. In the spirit of our other term changes that are moving to lowercase, I might be leaning toward it being lowercase, if the words are going to be separated. But again, I don't feel strongly, so I'm okay to go along with whatever is decided. Not sure if anyone else has any other thoughts CC @openshift/team-documentation

Reply:
I'm not sure about this -- GVK is particularly inside baseball, and quite relevant specifically for Operators, but mostly not for any other usage. (REST APIs as well.)

I don't know if it makes any sense as: _group version kind_. It's quite specifically API group, API version, API kind.

Reply:
Anywhere else, API references would simply be,

> x API group

or

> x API version

or just

> `NetworkPolicy` API / object

Reply:
> quite relevant specifically for Operators

This is why it's being added.

Reply:
I'm partial to _group/version/kind_, if we're allowed to get away with those space-less slashes.

Reply:
+1 to _group/version/kind_. Based on our other terminology updates, _GroupVersionKind_ would look like something that should be wrapped in back ticks, though it actually shouldn't be. :)

Reply:
> _GroupVersionKind_ would look like something that should be wrapped in back ticks, though it actually shouldn't be. :)

+1, this is actually how we have it currently and are trying to avoid. :sweat_smile:

Reply:
Updated to "group/version/kind" after team mtg and comments here.